pandas/tests/test_complex.py

```diff
@@ -0,0 +1,129 @@
+import numpy as np
```

Comment:
Any better locations for this test file?

Reply:
can you split the tests to the appropriate files: pandas/tests/series/methods/test_value_counts.py

for example